### PR TITLE
Clarify Windows (non-)support

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Since this is not clear to every user, clarify in the README that this collection does not support Windows targets.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ Provides modules for [Ansible](https://www.ansible.com/community) to manage [Mik
 
 You can find [documentation for the modules and plugins in this collection here](https://ansible.fontein.de/collections/community/routeros/).
 
+Please note that this collection does **not** support Windows targets for the modules not using the `network_cli` connection.
```
Comment (heuels) [inline README.md:8]: Sorry, I can't seem to grasp the meaning. **Windows target** here is a host machine on which Ansible is being run, right?

Comment (felixfontein) [reply README.md:8]: Target is the machine where the module is run on, when using the regular connection plugin. When using the network_cli connection, the modules are run locally (and connect to a network device), so for them there's technically no need to document this. But the `api` module is a regular module and - when not `delegate_to: localhost` is added or `localhost` is used - will run on whatever target the user is using (which cannot be a RouterOS device, but for example a jump host in a network which has access to the RouterOS device). Basically the above says "we do not support Windows jump hosts for the `community.routeros.api` module".

I guess we need to formulate this more clearly though :)

Comment (heuels) [reply README.md:8]: Got it, thanks! Should we maybe just straight-up say it then, I think this is perfect:

> ### Known issues / edge cases
> Please note that `community.routeros.api` module does **not** support Windows jump hosts!

Comment (heuels) [reply README.md]: It's great! 👍🏻

**Comment by felixfontein** [comment]
Zuul is currently broken, I'm therefore ignoring that negative check.

**Comment by felixfontein** [comment]
@heuels thanks for reviewing!

**Pull request merged**